File: Editorial/58_Length_of_Last_Word/C/3_Built_in_String_Functions.c
int lengthOfLastWord(char* s) {
    int count = 0;
    int len = strlen(s) - 1;
    // Discard the trailing spaces
    while (len >= 0 && s[len] == ' ') len--;
    // Iterate through the last word
    while (len >= 0 && s[len] != ' ') {
        count++;
        len--;
    }
    return count;
}

File: Editorial/0000-0100/0058_Length_of_Last_Word/Go/3_Built_in_String_Functions.go
func lengthOfLastWord(s string) int {
    count := 0
    lastIndex := len(s) - 1
    // Discard the trailing spaces
    for lastIndex >= 0 && s[lastIndex] == ' ' {
        lastIndex--
    }
    // Iterate through the last word
    for lastIndex >= 0 && s[lastIndex] != ' ' {
        count++
        lastIndex--
    }
    return count
}
